artiq/coredevice/ad53xx.py

```diff
@@ -366,3 +372,14 @@ def calibrate(self, channel, vzs, vfs):
         self.core.break_realtime()
         self.write_offset_mu(channel, 0x8000-offset_err)
         self.write_gain_mu(channel, 0xffff-gain_err)
+
+    @portable
+    def voltage_to_mu(self, voltage):
+        """Returns the 16-bit DAC register value required to produce a given
+        output voltage, assuming offset and gain errors have been trimmed out.
+
+        Valid voltages are: [-2*vref, + 2*vref - 1 LSB] + voltage offset
+
+        :param voltage: Voltage
```

Comment:
There is a return value you need to document as well. And the useful information for the input parameter is the unit: "SI Volts".